lib/Makefile

```diff
@@ -1,17 +1,10 @@
-all: hsmarker
-
-hsmarker: repo
-	mvn install -q -Dmaven.test.skip=true
-
-test:
-	mvn test -q
+all: repo
```

Comment:
infrastructure/infrastructure/pom.xml should also be edited to point to the new stanfordclassifier repo location